### ISSUE TITLE
Inline styles in component, we don't accept this code style.

### ISSUE THREAD

**Issue by AlexandrGordeev** [RoberttoHak/markdown-editor#1]
https://github.com/RoberttoHak/markdown-editor/blob/08e9f9c566ee64f03a7e4129a812f1165443d3fe/src/components/MarkdownEditor.vue#L124

For all places in code, we not accept styles in components. In bigger project we should not use style like this. It is make sence for this kind of project, but not more.

**Comment by RoberttoHak** [comment]
I agree with you, that's why I used them here :) I am ready not to use this way for larger projects
